template.go

```diff
@@ -11,36 +11,11 @@ import (
 	"text/template"
 )
 
-type OptionalString struct {
-	ptr *string
-}
-
-func (s OptionalString) String() string {
-	if s.ptr == nil {
-		return ""
-	}
-	return *s.ptr
-}
-
-func Require(arg interface{}) (string, error) {
-	if arg == nil {
+func Require(arg string) (string, error) {
+	if len(arg) == 0 {
 		return "", errors.New("Required argument is missing!")
```

Comment:
I'd change it to `Required argument is missing or empty!` but maybe once again i underestimate the user

Reply:
I changed it according to you.